src/osgPlugins/OpenFlight/FltExportVisitor.h

```diff
@@ -184,10 +184,10 @@ class FltExportVisitor : public osg::NodeVisitor
     typedef std::vector< osg::ref_ptr<osg::StateSet> > StateSetStack;
     StateSetStack _stateSetStack;
 
-    osg::ref_ptr<MaterialPaletteManager>     _materialPalette;
-    osg::ref_ptr<TexturePaletteManager>      _texturePalette;
-    osg::ref_ptr<LightSourcePaletteManager>  _lightSourcePalette;
-    osg::ref_ptr<VertexPaletteManager>       _vertexPalette;
+    std::auto_ptr<MaterialPaletteManager>     _materialPalette;
```

Comment:
Lets use the 3.6 version of this file as well. We don't have any 3.4 changes to this file.